Models/RepeatedProtoModel.cpp

```diff
@@ -105,7 +109,7 @@ QModelIndex RepeatedProtoModel::index(int row, int column, const QModelIndex & /
 
 Qt::ItemFlags RepeatedProtoModel::flags(const QModelIndex &index) const {
   if (!index.isValid()) return nullptr;
-  return QAbstractItemModel::flags(index);
+  return QAbstractItemModel::flags(index) | Qt::ItemIsEditable;
```

Comment:
Are we supposed to do this manually instead of telling our model it's editable? Do we always want these to be?

Reply:
It's easiest to just set the checkbox in the view to say its not editable